src/Raven.Server/ServerWide/Commands/ToggleDatabasesStateCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Raven.Server.Utils;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.ServerWide.Commands
+{
+    public class ToggleDatabasesStateCommand : UpdateValueCommand<ToggleParameters>
+    {
+        protected ToggleDatabasesStateCommand()
+        {
+            // for deserialization
+        }
+
+        public ToggleDatabasesStateCommand(ToggleParameters toggleParameters, string uniqueRequestId) : base(uniqueRequestId)
+        {
+            Value = toggleParameters;
+        }
+
+        public override object ValueToJson()
+        {
+            return Value.ToJson();
+        }
+
+        public override BlittableJsonReaderObject GetUpdatedValue(JsonOperationContext context, BlittableJsonReaderObject previousValue, long index)
+        {
+            return null;
+        }
+    }
+
+    public class ToggleParameters : IDynamicJson
+    {
+        public ToggleType ToggleType { get; set; }
+
+        public string[] DatabaseNames { get; set; }
+
+        public bool State { get; set; }
```

Comment:
I do not like that name, but I cannot think of a better one

Reply:
Changed to `Disable` to be more clear